### PR TITLE
add global header to replace header image

### LINKED ISSUE #122: Implement global navigation to replace the image header
## Description

The current global header is an image and better 

### Approach 1

Similar to kcov19's mobile navigation, we'll have a header w/ the current page name, as well as a bottom navigation.

image

### Approach 2

Mobile version of simple navbar from Tailwind UI

image

## Tasks

- [x] Find an SVG version of the WBW logo
- [x] #132 
- [x] Add app navigation

### PR DESCRIPTION
**Pull request by resir014**
## Description

Adds a header as well as global navigation for the entire website, as well as fixing the HTML hierarchy for all pages.

image

## Current Tasks

- [x] Global header
- ~~Global bottom navigation~~
- [x] HTML hierarchy fixes

### CODE REVIEW

File: tailwind.config.js [hunk 1]
```diff
@@ -5,6 +5,23 @@ module.exports = {
...
+        100: "#aed0f6",
+        200: "#80b5f1",
+        300: "#539beb",
+        400: "#2580e6",
+        500: "#1667c2",
+        600: "#114f94",
+        700: "#0c3666",
+        800: "#061e39",
+        900: "#01060b",
+      },
```
Comment (zainfathoni) [inline tailwind.config.js:23]: How did you generate these brand colours @resir014 ?

Comment (resir014) [reply tailwind.config.js:23]: I use https://www.tailwindshades.com/ - they support Tailwind 2.0 colours (up to `-50`)!

**Comment by resir014** [comment]
I'm trying to add the bottom navigation using list elements (`ul`, `li`), but it seems like the Google Docs styles override the navigation styles as well... What should we do with this, should we add the bottom navigation on a separate PR? @zainfathoni 

image

**Comment by zainfathoni** [comment]
Yes, please.
It's not _urgent_ at the moment.
Moreover, we will be decommissioning this page soon once #31 kicks in.

**Comment by resir014** [comment]
@zainfathoni Okay, this PR is ready then.

**Review by zainfathoni — APPROVED** [review]
LGTM :100:
Thanks, @resir014 !

**Pull request merged**